### DISCUSSION TITLE
A new Learn GraphQL course for the Svelte.js lovers! 📚

### DISCUSSION THREAD

**Discussion by mehtavishwa30** [hasura/graphql-engine#7279] [General] [answered]
Some benefits of **Svelte.js** that frontend developers enjoy are:

⚡️   **Better performance**
❰ ⁄ ❱ **Less boilerplate code**
💭   **Better state management**

We've put together a free Learn course for developers who want to combine the **power of GraphQL** with the **speed & performance of Svelte** to get up to speed with GraphQL and set it up for modern app development!

In this course, you will learn how to build faster apps that are also real-time & scalable by integrating #GraphQL APIs powered by Hasura with Svelte in less than 2-hours!
➡️ https://hasura.io/learn/graphql/svelte-apollo/introduction

**Comment by Adron** [comment, marked as answer]
**Good course!**

➡️ https://hasura.io/learn/graphql/svelte-apollo/introduction

*marking as answered* :)
